src/ansys/dpf/core/scoping.py

```diff
@@ -191,6 +194,9 @@ def _set_ids(self, ids: IdVectorType):
         """
         if isinstance(ids, range):
             ids = list(ids)
+        if isinstance(ids, np.ndarray):
+            if ids.dtype == np.int64:
```

Comment:
shouldn't we rather check the opposite: if not int32 -> convert to int32?

Reply:
Yes, can we have any other type of int in numpy? I mean, for sure the case of an array of floats shouldn't be handled, but any int should be handled as int32

Reply:
@PProfizi and what if I pass the array `np.array([0.2, 0.6, 5.0])`? That should raise, I think.